llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -33,7 +33,8 @@
 #include "llvm/Support/Debug.h"
 
 namespace {
```

Comment:
@AlexVlx Let's remove all references to `struct SyncScopeIDs` from the whole `SPIRVInstructionSelector.cpp` module. We will insert `getOrInsertSyncScopeID()` in the `getMemScope()` portion of code.

Reply:
Done.